README.md

```diff
@@ -128,11 +128,11 @@ The nf-core/proteinfold pipeline comes with documentation about the pipeline [us
 
 ## Credits
 
-nf-core/proteinfold was originally written by Athanasios Baltzis, Jose Espinosa-Carrasco, Luisa Santus, Harshil Patel.
+nf-core/proteinfold was originally written by Athanasios Baltzis ([@athbaltzis](https://github.com/athbaltzis)), Jose Espinosa-Carrasco ([@JoseEspinosa](https://github.com/JoseEspinosa)), Luisa Santus ([@luisas](https://github.com/luisas)), Harshil Patel ([@drpatelh](https://github.com/drpatelh)).
 
 We thank the following people for their extensive assistance in the development of this pipeline:
 
-<!-- TODO nf-core: If applicable, make list of people who have also contributed -->
+[@martin-steinegger](https://github.com/martin-steinegger), [@rjpbonnal](https://github.com/rjpbonnal), [@l-mansouri](https://github.com/l-mansouri)
```

Comment:
```suggestion
Many thanks to others who have helped out and contributed along the way too, including (but not limited to): [@martin-steinegger](https://github.com/martin-steinegger), [@rjpbonnal](https://github.com/rjpbonnal) and [@l-mansouri](https://github.com/l-mansouri)
```

Reply:
I will add with some more in a commit that I am preparing